src/core/reducer.test.js

```diff
@@ -341,6 +341,75 @@ test('undo / redo', () => {
   expect(state.G).toEqual({ A: true });
 });
 
+test('undo/redo phase', () => {
```

Comment:
Shall we group this test with the one above inside a `describe()`?

Reply:
Yes. I've grouped them together, and also simplified the test.